setup.py

```diff
@@ -195,6 +195,7 @@ def cython_extensions_and_necessity():
   extensions = [
       _extension.Extension(
           name=module_name,
+          language='c++',
```

Comment:
I might not be able to get a distrib test out in time for 1.3, but on a distribution built from this branch, it looks like we are OK

```
ldd py27/lib/python2.7/site-packages/grpc/_cython/cygrpc.so 
	linux-vdso.so.1 =>  (0x00007fffb67da000)
	librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f077687a000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f077665c000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f0776356000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f0776140000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f0775d78000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f0776ecd000)
```